src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/Views.ts

```diff
@@ -430,6 +464,9 @@ export var userProfileDirective = (
                 adhHttp.resolve(scope.path)
                     .then((res) => {
                         scope.userBasic = res.data[SIUserBasic.nick];
+                        getBadges(adhHttp, $q)(res).then((assignments) => {
+                            scope.assignments = assignments;
+                        });
```

Comment:
I would like to do that in another branch, as it effects all badges not just user badges.